src/etc/inc/interfaces.inc

```diff
@@ -3865,6 +3865,10 @@ function interface_dhcpv6_configure($interface = "wan", $wancfg) {
 		return;
 	}
 
+	if (file_exists("/conf/dhcp6c_duid")) { // Using user defined DUID.
+		exec("cp -f /conf/dhcp6c_duid /var/db/");
```

Comment:
PHP has a function to copy files, you should use copy() instead of a shell session